docs/src/details/xml/integration/arduino.rst

```diff
@@ -0,0 +1,155 @@
+.. _editor_integration_arduino:
+
+========================================
+Using the Exported C Code on Arduino IDE
+========================================
+
+Overview
+********
+
+This guide explains how to properly integrate your LVGL Editor exported project into 
+an Arduino sketch, why it must go inside the src folder, and how to fix include paths 
+for successful compilation.
+
+Arduino Build system
+--------------------
+
+Arduino uses a special build system that automatically compiles:
+- The main sketch file (``.ino``)
+- All ``.c``, ``.cpp``, and ``.h`` files located in the same directory as the sketch or inside the ``src`` subfolder.
+Any files placed outside these locations will not be compiled or linked automatically.
+
+Because LVGL Editor exports projects with nested folders like:
+- ``screens/``
+- ``widgets/``
+- ``components/``
+Arduino would ignore those files unless they’re inside ``src/``.
+That’s why placing your exported project under ``src/`` ensures everything is detected and compiled.
+
+Example structure
+-----------------
+
+.. code-block:: text
+
+    YourSketch/
+    ├── YourSketch.ino
+    └── src/
+          └── editor_project/
+              ├── editor_project.h
+              ├── editor_project.c
+              ├── screens/
+              ├── widgets/
+              └── components/
+
+Include paths
+-------------
+
+LVGL source files typically use this include pattern:
+
+.. code-block:: c
+
+    #ifdef LV_LVGL_H_INCLUDE_SIMPLE
+        #include "lvgl.h"
+    #else
+        #include "lvgl/lvgl.h"
+    #endif
+
+
+The ``LV_LVGL_H_INCLUDE_SIMPLE`` define allows flexibility in how LVGL is included.
+However, Arduino’s build system doesn’t provide a way to set this define globally across all source files.
+As a result, the compiler falls back to:
+
+.. code-block:: c
+
+    #include "lvgl/lvgl.h"
+
+which fails, because Arduino installs the LVGL library headers directly (e.g. ``libraries/lvgl/src/lvgl.h``), not in a nested ``lvgl/`` folder.
+
+To fix this, replace all occurrences of:
+
+.. code-block:: c
+
+    #include "lvgl/lvgl.h"
+
+with:
+
+.. code-block:: c
+
+    #include "lvgl.h"
```

Comment:
Question, is it not possible to locate the lv_conf on the Arduino case? I know for platform-io you can define this symbol in the project options, I think replacing the include paths (thing i did myself some cases) are error prone and impracticable in complex UI.